# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../avifile/ffmpeg-api.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur avifile-0.6-orig/lib/aviread/FFReadHandler.cpp avifile-0.6/lib/aviread/FFReadHandler.cpp
--- avifile-0.6-orig/lib/aviread/FFReadHandler.cpp	2008-08-30 12:00:40.000000000 +0200
+++ avifile-0.6/lib/aviread/FFReadHandler.cpp	2008-08-30 12:03:32.000000000 +0200
@@ -145,7 +145,7 @@
 int FFReadHandler::seek(framepos_t pos)
 {
     Locker locker(m_Mutex);
-    url_fseek(&m_pContext->pb, 0, SEEK_SET);
+    url_fseek(m_pContext->pb, 0, SEEK_SET);
     flush();
     //av_find_stream_info(m_pContext);
     return 0;
@@ -158,7 +158,7 @@
     AVM_WRITE("FF reader", 5, "readPacket()\n");
     if (av_read_packet(m_pContext, &pkt) < 0)
     {
-        if (!url_feof(&m_pContext->pb))
+        if (!url_feof(m_pContext->pb))
 	    AVM_WRITE("FF reader", "ffmpeg packet error and not eof??\n");
         return -1;
     }
